packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -194,6 +203,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
           this.transferFunds(wire);
           break;
         case PaidStreamingExtensionNotices.START:
+          // [ George ] Here we can call channelClient.createChannel()
```

Comment:
Based on the diagram by @spersico (see [here](https://whimsical.com/Sq6whAwa8aTjbwMRJc7vPU)), this might happen many times for a single peer + file / channel. So, probably not the right place to create the channel.